### PR TITLE
DEMO-133 Upload profile/ playlist image

### PR DESCRIPTION
**Pull request by Tantawit**
### Overview
As we discussed in our last meeting, create user and playlist will now required image file, these image will also attached into `getProfile` and `getPlaylist` response.

### Endpoint Changes
POST /users
**Body**:  form-data ⬇️
```
   email: string
   password: string
   profileImage: file
```

POST /users/playlists
**Body**:  form-data ⬇️
```
   name: string
   coverImage: file
   ... (other optional field that already exist)
```

### CODE REVIEW

File: src/users/dto/create-playlist.dto.ts [hunk 1]
```diff
@@ -17,12 +17,10 @@ export class CreatePlaylistDto {
...
-  @ApiProperty({ example: 'https://i.imgur.com/1J2h9YR.jpg' })
-  @IsNotEmpty()
-  @IsString()
-  coverImage: string;
-
   @ApiProperty({ example: true })
   @IsOptional()
   isAlbum: boolean;
+
+  @ApiProperty({ example: 'please upload image file in this field' })
```
Comment (pol-alones) [inline src/users/dto/create-playlist.dto.ts]: I think there is an option about `description`

File: src/users/dto/create-user.dto.ts [hunk 2]
```diff
@@ -12,6 +12,9 @@ export class CreateUserDto {
   @IsNotEmpty()
   @IsEmail()
   email: string;
+
+  @ApiProperty({ example: 'please upload image file in this field' })
```
Comment (pol-alones) [inline src/users/dto/create-user.dto.ts]: I think there is an option about `description`

**Review by pol-alones — APPROVED** [review]
Nice, but change the `example` to be `description`

**Review by sseerrnn — APPROVED** [review]

File: src/users/dto/create-playlist.dto.ts (hunk 1 above)
Comment (Tantawit) [reply src/users/dto/create-playlist.dto.ts]: I think it provide more readablity this way, since it wil show my message in the example value.
image

Comment (Tantawit) [reply src/users/dto/create-playlist.dto.ts]: This is now as discussed in our last meeting.

File: src/users/dto/create-user.dto.ts (hunk 2 above)
Comment (Tantawit) [reply src/users/dto/create-user.dto.ts]: This is now as discussed in our last meeting.

**Pull request merged**